src/test/java/org/opensearch/security/securityconf/SecurityRolesPermissionsTest.java

```diff
@@ -171,7 +186,7 @@ public void hasExplicitClusterPermissionPermissionForRestAdmin() {
         // verify all endpoint except SSL
         final Collection<Endpoint> noSslEndpoints = ENDPOINTS_WITH_PERMISSIONS.keySet()
             .stream()
-            .filter(e -> e != Endpoint.SSL)
+            .filter(e -> e != Endpoint.SSL && e != Endpoint.CONFIG)
```

Comment:
Why add this check here? Also, the comment above should be updated if we decide to keep this.

Reply:
Should we modify the comment on line 186?

Reply:
done